pkg/loop/internal/relayer/pluginprovider/chainreader/chain_reader_test.go

```diff
@@ -287,6 +287,18 @@ func (it *fakeChainReaderInterfaceTester) GetChainReader(_ *testing.T) types.Con
 	return it.impl
 }
 
+func (it *fakeChainReaderInterfaceTester) StartChainReader(t *testing.T) {
+	fake, ok := it.impl.(*fakeChainReader)
+	assert.True(t, ok)
+	require.NoError(t, fake.Start(context.Background()))
+}
+
+func (it *fakeChainReaderInterfaceTester) CloseChainReader(t *testing.T) {
```

Comment:
Could these be called just `Start` and `Close`? The type already says `ChainReader`
```suggestion
func (it *fakeChainReaderInterfaceTester) Start(t *testing.T) {
	fake, ok := it.impl.(*fakeChainReader)
	assert.True(t, ok)
	require.NoError(t, fake.Start(context.Background()))
}

func (it *fakeChainReaderInterfaceTester) Close(t *testing.T) {
```

Reply:
Thanks for this! Renamed here: [`8e1388b` (#705)](https://github.com/smartcontractkit/chainlink-common/pull/705/commits/8e1388bc2819bbd99eed8831e18286e9ad49860f)